redis/src/aio/connection.rs

```diff
@@ -224,9 +244,7 @@ where
             }
             loop {
                 match self.read_response().await? {
-                    Value::Push { .. } => {
-                        //self.execute_push_message(kind, data) //TODO
-                    }
+                    Value::Push { .. } => continue,
```

Comment:
why not handle PUSH here, instead of in `read_response`?

Reply:
can you please explain it to me? 
I don't understand - why have 2 branches over PUSH instead of 1? We have a place handling PUSH logic, why not handle everything here?
It's not a big deal, I just don't understand the intention here.

Reply:
OK, I think I see a reason - otherwise you need the same logic twice, here and in `req_packed_commands` (which still needs to be fixed).
BTW, please find and replace `execute_push_message` throughout the repo. There's connections that need handling.